test/worlds/models/building_L1/meshes/model.material

```diff
@@ -0,0 +1,26 @@
+material floor_1_Diffuse
```

Comment:
Ignition doesn't support Ogre material files, so it's strange to have these files here on test cases. I think it would be best to remove these so users don't get the idea that there's some support for this.

Reply:
addressed in this PR https://github.com/ignitionrobotics/ign-gazebo/pull/923 along with another fix